src/main/webapp/script.js

```diff
@@ -103,13 +103,21 @@ function createCommentElement(comment) {
   commentElement.className = 'small-sep';
 
   const userComment = document.createElement('span');
-  var userInfoDisplayed = comment.username + " • " + comment.location + " • " + comment.MMDDYYYY;
+  // Username hyperlinks to the profile of the user who left the comment.
+  // If there is no associated user (i.e. anon commenter), will just link to an invalid page.
```

Comment:
You could also just not make the "anonymous" name be a link. That might be preferable, what do you think?

Reply:
Done